mysql-test/main/func_kdf.test

```diff
@@ -55,3 +55,13 @@ select length(kdf('foo', 'bar', 100, 'pbkdf2_hmac', 65537));
 --echo # End of 11.3 tests
 --echo #
 
+--echo #
+--echo # Test kdf() without parameters
```

Comment:
Last request - put the MDEV number at the beginning of the `echo` - same with other tests.